packages/tendermint-rpc/src/config.spec.ts

```diff
@@ -1,7 +1,17 @@
+export interface ExpectedValues {
+  readonly appCreator: string;
+  readonly p2pVersion: number;
+  readonly blockVersion: number;
+  readonly appVersion: number;
+}
+
 export interface TendermintInstance {
   readonly url: string;
   readonly version: string;
-  readonly appCreator: string;
+  /** Values we expect in the backend */
+  readonly expected: ExpectedValues;
+  /** A workaround for https://github.com/tendermint/tendermint/issues/4489 */
```

Comment:
😬

Reply:
Fortunately the clock issue is relsolved now